README.md

```diff
@@ -16,6 +16,21 @@ then import the jQuery package:
 import io.udash.wrappers.jquery._
 ```
 
+Since version `2.0.0` the wrapper does not force JS dependency on jQuery. You have to 
+add it manually by:
+ * explicit link in your `index.html`.
+ ```html
+ <script src="https://code.jquery.com/jquery-3.3.1.min.js" 
+         integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" 
+         crossorigin="anonymous"></script>
+ ```
+ * [Scala.js dependency](http://www.scala-js.org/doc/project/dependencies.html).
```

Comment:
"or a"? If I didn't know, I'd add both. Also the jsDependency variant is not equal to adding a script in html - you still need to add a script tag or load this from JS.

Reply:
The linked Scala.js doc describes how to work with js deps. I think we don't need to copy it here.